### PR TITLE
Fixes #24740: document --watch-poll for NFS

### PR DESCRIPTION
**Pull request by akofink**
From https://community.theforeman.org/t/webpack-watch-over-nfs/10922/2

### CODE REVIEW

**Review by ekohl — DISMISSED** [review]

**Comment by lizagilman** [comment]
Failing tests unrelated? 
They seem to fails in some other pr's as well

**Comment by akofink** [comment]
I'd hope a readme change wouldn't cause tests to fail. ;)

**Comment by lizagilman** [comment]
@akofink oops didn't notice :rofl:

File: webpack/stories/docs/gettingStarted.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ Following steps are required to setup a webpack development environment:
       --https \
       --key /etc/pki/katello/private/katello-apache.key \
       --cert /etc/pki/katello/certs/katello-apache.crt \
-      --cacert /etc/pki/katello/certs/katello-default-ca.crt
+      --cacert /etc/pki/katello/certs/katello-default-ca.crt\
```
Comment (mmoll) [inline webpack/stories/docs/gettingStarted.md]: please add a space before the backslash

**Review by mmoll — CHANGES_REQUESTED** [review]

**Comment by akofink** [comment]
Updated

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @akofink!

**Pull request merged**